docs/node/run-your-node/advanced/using-ledger-backed-consensus-key-with-a-remote-signer.md

```diff
@@ -22,7 +22,7 @@ Before following this guide, make sure you've followed the [Prerequisites](../pr
 
 Also make sure you've followed the [Install Oasis Remote Signer Binary](install-oasis-remote-signer-binary.md) section and have the Oasis Remote Signer binary installed on your remote signer system where your Ledger wallet will be connected.
 
-Finally, you need to do steps described in the [Setup](../../../oasis-core-ledger/usage/setup.md) section of our [Oasis Core Ledger docs](../../../oasis-core-ledger). Make sure you also install the _**OasisVal**_** app** on your Ledger wallet.
+Finally, make sure you install the _**OasisVal**_** app** on your Ledger wallet.
```

Comment:
@kostko Does simply installing OasisVal app suffice or do we need something else? The existing oasis-core-ledger Setup section mentions something about the ["Ledger signing plugin"](https://github.com/oasisprotocol/oasis-core-ledger/blob/master/docs/usage/setup.md#ledger-signer-plugin), but this has been deprecated in 23.0 afaik.

Reply:
I would change this entire chapter to just speak about the remote signer setup but without the Ledger stuff. Because yeah, using a Ledger requires the use of the legacy "Ledger signing plugin".

Reply:
Done. Please re-review.